pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -74,7 +76,12 @@ func totalImageSize(nodeInfo *schedulercache.NodeInfo, containers []v1.Container
 
```

Comment:
We should add a TODO comment here to describe the conner cases we did not cover right now.

In my mind, the exceptions are:

1. If use Docker as runtime, i.e. `docker.io/library/test:tag` in pod spec but only `test:tag` will present in node status.
2. First present implicit registry in pod spec. i.e. `test:tag` in pod spec but only `docker.io/library/test:tag` in node status. Please note this should not happen when user consistently use one type of registry format.